modules/customizing-cli-downloads.adoc

```diff
@@ -15,6 +15,24 @@ the packages.
 
 . Select *ConsoleCLIDownload* from the list of Custom Resource Definitions (CRDs).
 
-. Click the *YAML* tab, and then make your edits.
+. Click the *YAML* tab, and then make your edits:
++
+----
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: example-cli-download-links-for-foo.
+spec:
+  description: |
+    This is an example of download links for foo
+  displayName: example-foo
+  links:
+    - href: 'https://www.foo-for-linux.com'
```

Comment:
Its prob a good idea to imply that these are individual files, so this might be better example text:

```yaml
- href: 'https://www.example.com/public/foo.tar'
  text: foo for linux
- href: 'https://www.example.com/public/foo.mac.zip'
  text: foo for mac
- href: 'https://www.example.com/public/foo.win.zip'
  text: foo for windows
```